File: PHP/Pakaian.php
<?php

require ('Barang.php');

class Pakaian extends Barang{

    private $ukuran = '';
    private $bahan = '';
    private $gender = '';

    public function __construct( $ukuran = '', $bahan = '', $gender = ''){
        $this->ukuran = $ukuran;
        $this->bahan = $bahan;
        $this->gender = $gender;
    }

    public function setukuran($ukuran){
        $this->ukuran = $ukuran;
    }

    public function setbahan($bahan){
        $this->bahan = $bahan;
    }

    public function setgender($gender){
        $this->gender = $gender;
    }



    public function getukuran(){
        return $this->ukuran;
    }

    public function getbahan(){
        return $this->bahan;
    }

    public function getgender(){
        return $this->gender;
    }

    
}

?>

File: Java/Pakaian.java
class Pakaian extends Barang{
    private String ukuran;
    private String bahan;
    private String gender;
 

    Pakaian() {

    }

    Pakaian(String ukuran, String bahan, String gender){
        this.ukuran = ukuran;
        this.bahan = bahan;
        this.gender = gender;
    }

    public void setukuran(String ukuran){
        this.ukuran = ukuran;
    }

    public void setbahan(String bahan){
        this.bahan = bahan;
    }

    public void setgender(String gender){
        this.gender = gender;
    }

    public String getukuran() {
        return this.ukuran;
    }

    public String getbahan() {
        return this.bahan;
    }

    public String getgender() {
        return this.gender;
    }

}


